lib/src/settings.rs

```diff
@@ -159,7 +159,7 @@ impl UserSettings {
             // For compatibility with old config files (<0.8.0)
             self.config
                 .get_string("ui.default-revset")
-                .unwrap_or_else(|_| "@ | ancestors((remote_branches() | tags()).., 2)".to_string())
+                .unwrap_or_else(|_| "@ | ancestors(immutable().., 2)".to_string())
```

Comment:
(Not for this PR): I'm thinking about moving all these defaults to the TOML files, so that the end-user can see them with `jj config get`. Ultimately, `jj config get` could be smart enough to debug the config (say whether the user is using the default or something overrode it). I think we could just panic if they aren't provided.

Reply:
I agree. It's nice to be able to see as much of the default configs as possible with `jj config list --include-defaults`